aiomonitor/monitor.py

```diff
@@ -43,17 +44,26 @@ def __init__(self, cmds: List['CmdName']) -> None:
         super().__init__()
 
 
+class ArgumentMappingException(CommandException):
+    pass
+
+
 CmdName = NamedTuple('CmdName', [('cmd_name', str), ('method_name', str)])
 
 
 class Monitor:
     _event_loop_thread_id = None  # type: int
     _cmd_prefix = 'do_'
+    _empty_result = object()
 
     prompt = 'monitor >>> '
+    intro = '\nAsyncio Monitor: {tasknum} task{s} running\nType help for available commands\n\n'  # noqa
     help_template = '{cmd_name} {arg_list}\n    {doc}\n'
     help_short_template = '    {cmd_name}{cmd_arg_sep}{arg_list}: {doc_firstline}'  # noqa
 
+    _sin = None  # type: IO[str]
```

Comment:
probably `Optional[IO[str]]`

Reply:
This gave me problems further down the road, where `mypy` does not realise that within the `do_*` methods, `_sout` is guaranteed to be set to an instance of `IO[str]`, so it errors out because `None` doesn't have a `write` method.